singularity-eos/eos/eos.hpp

```diff
@@ -63,7 +64,7 @@ using singularity::detail::transform_variadic_list;
 
 // all eos's
 static constexpr const auto full_eos_list =
-    tl<IdealGas, Gruneisen, Vinet, JWL, DavisReactants, DavisProducts
+    tl<IdealGas, Gruneisen, Vinet, JWL, DavisReactants, DavisProducts, StiffGas
```

Comment:
@jhp-lanl you mentioned this afternoon that we might want to move this EOS to be in the variant only optionally?

Reply:
1. Testing still to be done on Darwin
2. Added PR to changelog
3. Documentation is still lacking, but a brief report is already done in LaTeX format - to be passed on or put in sphinx eventually.
4. Code formatted

Reply:
I would propose we keep this in the typelist for the Variant since it's going to be used in xRAGE.